articles/openshift/oshift-trials-gs.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Getting Started Guide for UKCloud for OpenShift trials| UKCloud Ltd
+description: Provides information to get up and running with UKCloud for OpenShift trials
+services: openshift
+author: Mudasar Hussain
+
+toc_rootlink: Getting Started
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: UKCloud for OpenShift
+toc_fullpath: Getting Started/oshift-trials-gs.md
+toc_mdlink: oshift-trials-gs.md
+---
+
+# Getting Started Guide for UKCloud for OpenShift trials
+
+## Overview
+
+Welcome to your Red Hat&trade; OpenShift on UKCloud trial. Your trial OpenShift cluster is a secure, single-tenant private deployment on UKClouds assured OpenStack cloud which avoids any noisy neighbour issues. During an OpenShift trial the OpenShift cluster is typically only connected to the internet, however connectivity to many other Government Community Networks is available in OpenShift clusters post trial phase. The purpose of an OpenShift trial is for customers to get hands on with Red Hat&trade; OpenShift container orchestration technology to see if this is the correct technology for them.
```

Comment:
Suggest (asterisks/bold indicate changes - don't actually put the asterisks in):

Welcome to your *UKCloud for OpenShift* trial. Your trial OpenShift cluster is a secure, single-tenant private deployment on *UKCloud's* assured OpenStack cloud*,* which avoids any noisy neighbour issues. During an OpenShift trial the OpenShift cluster is typically only connected to the internet, however connectivity to many other Government Community Networks is available in OpenShift clusters *after the* trial phase. The purpose of an OpenShift trial is for *you* to get *hands-on experience* with Red Hat&trade; OpenShift container orchestration technology to see if this is the correct technology for *you*.